src/Microsoft.ML.Transforms/CategoricalsCatalog.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms.Categorical;
+
+namespace Microsoft.ML
+{
+
+    /// <summary>
+    /// Extensions for KeyToVectorMappingEstimator.
+    /// </summary>
+    public static class ToBinaryVectorCatalog
+    {
+        public static KeyToBinaryVectorMappingEstimator ToBinaryVector(this TransformsCatalog.CategoricalTransforms catalog, params KeyToBinaryVectorTransform.ColumnInfo[] columns)
+        => new KeyToBinaryVectorMappingEstimator(CatalogUtils.GetEnvironment(catalog), columns);
+
+        public static KeyToBinaryVectorMappingEstimator ToBinaryVector(this TransformsCatalog.CategoricalTransforms catalog, string name, string source = null)
```

Comment:
>string name, string source = null [](start = 125, length = 33)

Did we change the naming convention from name/source to inputColumn/outputColumn? Or name/source is still valid?

Also I noticed, that some some of the extensions have output = null, as optional and some don't.

Reply:
We are going with InputColumn outputColumn. I'll scan for inconsistencies. Thanks for pointing out. 

---
In reply to: [229710031](https://github.com/dotnet/machinelearning/pull/1460#discussion_r229710031) [](ancestors = 229710031)